public/data/RightContainer.json

```
@@ -0,0 +1,87 @@
+{
+  "result": {
+    "product": [
+      {
+        "id": "1",
+        "image_url": "https://images.unsplash.com/photo-1491553895911-0055eca6402d?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1760&q=80",
+        "brand": "NIke",
+        "en_name": "Nike Dunk Low Retro Black",
+        "ko_name": "나이키 덩크 로우 레트로 블랙",
+        "buy_now_price": "284,000원",
```

```suggestion
        "buy_now_price": 284000,
```
string이 아니라 number가 되어야 하지 않나요?

---

바로 수정하겠습니다 !!!!

---

수정했습니다!!

---

@jaewoo-ahn 수정 안된거 같습니다~